functional-tests/tests/bridge_withdraw_happy_op_return.py

```diff
@@ -0,0 +1,115 @@
+    and if the BTC has an OP_RETURN block.
+    """
+
+    def __init__(self, ctx: flexitest.InitContext):
+        fast_batch_settings = net_settings.get_fast_batch_settings()
+        ctx.set_env(
+            testenv.BasicEnvConfig(
+                pre_generate_blocks=101,
+                rollup_settings=fast_batch_settings,
+            )
+        )
+
+    def main(self, ctx: flexitest.RunContext):
+        btc = ctx.get_service("bitcoin")
+        seq = ctx.get_service("sequencer")
+        # create both btc and sequencer RPC
+        btcrpc: BitcoindClient = btc.create_rpc()
+        seqrpc = seq.create_rpc()
+
+        # Wait for seq
+        wait_until(
+            lambda: seqrpc.strata_protocolVersion() is not None,
+            error_with="Sequencer did not start on time",
+        )
+
+        # Generate addresses
+        address = ctx.env.gen_ext_btc_address()
+        withdraw_address = ctx.env.gen_ext_btc_address()
+        el_address = self.eth_account.address
+        payload = "hello world"
+        bosd = string_to_opreturn_descriptor(payload)
+        self.debug(f"BOSD: {bosd}")
+
+        self.debug(f"Address: {address}")
+        self.debug(f"Change Address: {withdraw_address}")
+        self.debug(f"EL Address: {el_address}")
+
+        # Original BTC balance
+        btc_url = self.btcrpc.base_url
+        btc_user = self.btc.get_prop("rpc_user")
+        btc_password = self.btc.get_prop("rpc_password")
+        original_balance = get_balance(withdraw_address, btc_url, btc_user, btc_password)
+        self.debug(f"BTC balance before withdraw: {original_balance}")
+
+        # Make sure starting ETH balance is 0
+        check_initial_eth_balance(self.rethrpc, el_address, self.debug)
+
+        bridge_pk = get_bridge_pubkey(self.seqrpc)
+        self.debug(f"Bridge pubkey: {bridge_pk}")
+
+        # make two deposits
+        self.deposit(ctx, el_address, bridge_pk)
+        self.deposit(ctx, el_address, bridge_pk)
+
+        # Withdraw
+        _, withdraw_tx_receipt, _ = self.withdraw(ctx, el_address, bosd)
+
+        # Collect L2 and L1 blocks where the withdrawal has happened.
+        l2_withdraw_block_num = withdraw_tx_receipt["blockNumber"]
+        self.info(f"withdrawal block num on L2: {l2_withdraw_block_num}")
+
+        last_block_hash = btcrpc.proxy.getblockchaininfo()["bestblockhash"]
+        last_block = btcrpc.getblock(last_block_hash)
+        # Check all blocks down from the latest.
+        # Those blocks will have only coinbase tx for all the empty blocks.
+        # Block with the withdrawal transfer will have at least two transactions.
+        l1_withdraw_block_height = last_block["height"]
+        self.info(f"withdrawal block height on L1: {l1_withdraw_block_height}")
+
+        return True
+
+        # FIXME: Somehow the block does not have the "hello world"
+        # (OP_RETURN LEN 68656c6c6f20776f726c64)
+        # payload. I think that this will be fixed once we support
```

Comment:
This sounds weird. How though? because then we would be looking at all the outputs?

Reply:
To be fair, I desperately need some help and ideas on how to make the `bridge_withdraw_happy_op_return` work 😓

Reply:
Light told me yesterday that any `OP_RETURN` <=80bytes is standard even with amounts tied to it...